frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/configure.tsx

```diff
@@ -91,6 +92,12 @@ export const EncryptionFormGroup: React.FC<EncryptionFormGroupProps> = ({
   };
 
   const toggleAdvancedEncryption = (checked: boolean) => {
+    let kmsObject: KMSConfig;
+
+    if (!checked) {
+      kmsObject = initialState.kms;
+      setDispatch(ActionType.SET_KMS_ENCRYPTION, kmsObject, mode, dispatch);
```

Comment:
I would prefer to call it following, just to avoid confusion with root state dispatcher
```suggestion
      setKms(ActionType.SET_KMS_ENCRYPTION, kmsObject, mode, dispatch);
```